week1/homework/src/server.js

```diff
@@ -1,20 +1,49 @@
+
 'use strict';
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
 
-  const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+  const server = http.createServer((req, response) => {
+    function printData() {
+      response.writeHead(200, {
+        'Content-Type': 'application/json'
+      });
+      let json = JSON.stringify({
```

Comment:
you can instead of defining the variable separately use it in the function call 

This is called to inline the variable (see what they say inlining is here: https://blog.jetbrains.com/webstorm/2009/10/javascript-refactoring-inline-variable )

Reply:
I think I got your point.